Impossible to use `opentelekomcloud_vpc_peering_connection_v2`

<!--
Hi there, thank you for opening an issue.
-->

# Affected Resource(s)
<!--
Please list the resources as a list, for example:
- opentelekomcloud_compute_instance_v2
- opentelekomcloud_blockstorage_volume_v2
-->
- `opentelekomcloud_vpc_peering_connection_v2`

# Terraform Configuration Files
```hcl
resource "opentelekomcloud_vpc_v1" "vpc_1" {
  name = "vpc_test"
  cidr = "192.168.0.0/16"
}

resource "opentelekomcloud_vpc_v1" "vpc_2" {
  name = "vpc_test1"
  cidr = "192.168.0.0/16"
}

resource "opentelekomcloud_vpc_peering_connection_v2" "peering_1" {
  name        = "opentelekomcloud_peering"
  vpc_id      = opentelekomcloud_vpc_v1.vpc_1.id
  peer_vpc_id = opentelekomcloud_vpc_v1.vpc_2.id
}

```

# Debug Output/Panic Output
```
panic: Invalid address to set: []string{"id"}

goroutine 484 [running]:
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*ResourceData).Set(0xc0007f5300, 0x1752970, 0x2, 0x14c7c00, 0xc000b546c0, 0x149d800, 0xc0009326c0)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/resource_data.go:230 +0x371
github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/services/vpc.resourceVPCPeeringV2Read(0x19e9698, 0xc0007e5da0, 0xc0007f5300, 0x1748520, 0xc00054c000, 0x0, 0x0, 0x0)
	/home/akachurin/GolandProjects/terraform-provider-opentelekomcloud/opentelekomcloud/services/vpc/resource_opentelekomcloud_vpc_peering_connection_v2.go:136 +0x395
github.com/opentelekomcloud/terraform-provider-opentelekomcloud/opentelekomcloud/services/vpc.resourceVPCPeeringV2Create(0x19e9698, 0xc0007e5da0, 0xc0007f5300, 0x1748520, 0xc00054c000, 0xc000b85b30, 0x7b84aa, 0xc001203620)
	/home/akachurin/GolandProjects/terraform-provider-opentelekomcloud/opentelekomcloud/services/vpc/resource_opentelekomcloud_vpc_peering_connection_v2.go:115 +0x7ea
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0xc0009867e0, 0x19e9628, 0xc0002faec0, 0xc0007f5300, 0x1748520, 0xc00054c000, 0x0, 0x0, 0x0)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/resource.go:330 +0x17f
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0xc0009867e0, 0x19e9628, 0xc0002faec0, 0xc0005aad20, 0xc001203620, 0x1748520, 0xc00054c000, 0x0, 0x0, 0x0, ...)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/resource.go:456 +0x67b
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0xc0008f29d8, 0x19e9628, 0xc0002faec0, 0xc00089b400, 0xc0002faec0, 0x16e01a0, 0xc000b8a600)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-sdk/v2@v2.7.0/helper/schema/grpc_provider.go:955 +0x8ef
github.com/hashicorp/terraform-plugin-go/tfprotov5/server.(*server).ApplyResourceChange(0xc00059cc20, 0x19e96d0, 0xc0002faec0, 0xc0005aabd0, 0xc00059cc20, 0xc000b8a6c0, 0xc0007a3ba0)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/server/server.go:332 +0xb5
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler(0x16e01a0, 0xc00059cc20, 0x19e96d0, 0xc000b8a6c0, 0xc0007e59e0, 0x0, 0x19e96d0, 0xc000b8a6c0, 0xc0005b8900, 0x20f)
	/home/akachurin/go/pkg/mod/github.com/hashicorp/terraform-plugin-go@v0.3.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:380 +0x214
google.golang.org/grpc.(*Server).processUnaryRPC(0xc00063e000, 0x19f56d8, 0xc001014000, 0xc000ada600, 0xc000b531a0, 0x2288c00, 0x0, 0x0, 0x0)
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:1194 +0x52b
google.golang.org/grpc.(*Server).handleStream(0xc00063e000, 0x19f56d8, 0xc001014000, 0xc000ada600, 0x0)
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:1517 +0xd0c
google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc00015a0e0, 0xc00063e000, 0x19f56d8, 0xc001014000, 0xc000ada600)
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:859 +0xab
created by google.golang.org/grpc.(*Server).serveStreams.func1
	/home/akachurin/go/pkg/mod/google.golang.org/grpc@v1.32.0/server.go:857 +0x1fd


Process finished with the exit code 1


```

# Steps to Reproduce
<!--
Please list the steps required to reproduce the issue, for example:
-->
1. `terraform apply`

# Important Factoids
<!--
Are there anything atypical about your accounts that we should know?
For example: Which version of OpenTelekomCloud? Tight ACLs.
-->
Found running acceptance test `TestAccOTCVpcRouteV2DataSource_basic`


Blocks #1240

--- a/opentelekomcloud/services/vpc/data_source_opentelekomcloud_vpc_peering_connection_v2.go
+++ b/opentelekomcloud/services/vpc/data_source_opentelekomcloud_vpc_peering_connection_v2.go
@@ -4,6 +4,7 @@ import (
 	"context"
 	"log"
 
+	"github.com/hashicorp/go-multierror"
 	"github.com/opentelekomcloud/gophertelekomcloud/openstack/networking/v2/peerings"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
@@ -89,13 +90,17 @@ func dataSourceVpcPeeringConnectionV2Read(_ context.Context, d *schema.ResourceD
 	log.Printf("[INFO] Retrieved Vpc peering Connections using given filter %s: %+v", Peering.ID, Peering)
 	d.SetId(Peering.ID)
 
-	d.Set("id", Peering.ID)
-	d.Set("name", Peering.Name)
-	d.Set("status", Peering.Status)
-	d.Set("vpc_id", Peering.RequestVpcInfo.VpcId)
-	d.Set("peer_vpc_id", Peering.AcceptVpcInfo.VpcId)
-	d.Set("peer_tenant_id", Peering.AcceptVpcInfo.TenantId)
-	d.Set("region", config.GetRegion(d))
+	mErr := multierror.Append(
+		d.Set("name", Peering.Name),
+		d.Set("status", Peering.Status),
+		d.Set("vpc_id", Peering.RequestVpcInfo.VpcId),
+		d.Set("peer_vpc_id", Peering.AcceptVpcInfo.VpcId),
+		d.Set("peer_tenant_id", Peering.AcceptVpcInfo.TenantId),
+		d.Set("region", config.GetRegion(d)),
+	)
+	if err := mErr.ErrorOrNil(); err != nil {
+		return fmterr.Errorf("error setting VPC peering attributes: %w", err)
+	}
 
 	return nil
 }

--- a/opentelekomcloud/services/vpc/resource_opentelekomcloud_vpc_peering_connection_v2.go
+++ b/opentelekomcloud/services/vpc/resource_opentelekomcloud_vpc_peering_connection_v2.go
@@ -5,6 +5,7 @@ import (
 	"log"
 	"time"
 
+	"github.com/hashicorp/go-multierror"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
@@ -133,13 +134,17 @@ func resourceVPCPeeringV2Read(_ context.Context, d *schema.ResourceData, meta in
 		return fmterr.Errorf("error retrieving OpenTelekomCloud Vpc Peering Connection: %s", err)
 	}
 
-	d.Set("id", n.ID)
-	d.Set("name", n.Name)
-	d.Set("status", n.Status)
-	d.Set("vpc_id", n.RequestVpcInfo.VpcId)
-	d.Set("peer_vpc_id", n.AcceptVpcInfo.VpcId)
-	d.Set("peer_tenant_id", n.AcceptVpcInfo.TenantId)
-	d.Set("region", config.GetRegion(d))
+	mErr := multierror.Append(
+		d.Set("name", n.Name),
+		d.Set("status", n.Status),
+		d.Set("vpc_id", n.RequestVpcInfo.VpcId),
+		d.Set("peer_vpc_id", n.AcceptVpcInfo.VpcId),
+		d.Set("peer_tenant_id", n.AcceptVpcInfo.TenantId),
+		d.Set("region", config.GetRegion(d)),
+	)
+	if err := mErr.ErrorOrNil(); err != nil {
+		return fmterr.Errorf("error setting VPC peering attributes: %w", err)
+	}
 
 	return nil
 }

--- a/releasenotes/notes/vpc-peering-d5f0618eeb9fa55e.yaml
+++ b/releasenotes/notes/vpc-peering-d5f0618eeb9fa55e.yaml
@@ -1,0 +1,4 @@
+---
+fixes:
+  - |
+    **[VPC]** Fix panic when using ``resource/opentelekomcloud_vpc_peering_connection_v2`` and ``data_source/opentelekomcloud_vpc_peering_connection_v2`` (`#1242 <https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/pull/1242>`_)

